### ISSUE TITLE
cordova-plugin-compat No Longer Required

### ISSUE THREAD

**Issue by vestaxpdx** [Qbix/cordova-plugin-browsertabs#3]
Hi

Forgive me if I am wrong, but I am getting this error when adding the plugin for android.

```
Plugin doesn't support this project's cordova-android version. cordova-android: 8.1.0, failed version requirement:
      <6.3.0
Skipping 'cordova-plugin-compat' for android
```

Acording to here: https://cordova.apache.org/announcements/2017/09/27/android-release.html cordova-plugin-compat is no longer needed so the dependency can be removed from in cordova-plugin-browsertab/plugin.xml ?

**Comment by phuwin** [comment]
I think you are at the wrong repo.

**Comment by vestaxpdx** [comment]
I don't think so. If I remove line 13 from cordova-plugin-browsertab/plugin.xml the warning is not shown when you build and everything works fine.

```
<dependency id="cordova-plugin-compat" version="^1.0.0" />
```
